app/components/DepartureTimeHr.tsx

```diff
@@ -6,9 +6,11 @@ function DepartureTimeHr(props: { departure: any }) {
     const departureTime = new Date(departureDateTime).toLocaleTimeString('fr-EU', { hour: '2-digit', minute: '2-digit' })
 
     return (
-        <div>
+        departureTime
+        ? (<div>
             {departureTime}    
-        </div>
+        </div>)
+        : (<p>??:??</p>)
```

Comment:
In this case the parent should have prevented the component from rendering 
Also departureTime will never be null, it'll be a date as you wanted or an `Invalid Date`

Reply:
I propose to use this to check the validity of the Date : 
`Date.parse(departureDateTime)`